packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -85,11 +72,14 @@ export function HorizontalCellDivider(): JSX.Element {
 
 export class ColumnHeaderCell extends AbstractPureComponent<IColumnHeaderCellProps, IColumnHeaderCellState> {
     public static defaultProps: IColumnHeaderCellProps = {
-        enableColumnInteractionBar: false,
         isActive: false,
         menuIconName: "chevron-down",
     };
 
+    public static contextTypes: React.ValidationMap<
+        IColumnInteractionBarContextTypes
+    > = columnInteractionBarContextTypes;
```

Comment:
shouldn't need `React.ValidationMap...` type info. let it come from `columnInteractionBarContextTypes`

Reply:
you do need it. there's a whole "typescript cannot name this type even though it knows where it comes from" rabbit hole which I won't bother explaining here

Reply:
dang i gotcha